components/server/src/projects/projects-service.ts

```diff
@@ -148,7 +148,7 @@ export class ProjectsService {
         const parsedUrl = RepoURL.parseRepoUrl(project.cloneUrl);
         const hostContext = parsedUrl?.host ? this.hostContextProvider.get(parsedUrl?.host) : undefined;
         const type = hostContext && hostContext.authProvider.info.authProviderType;
-        if (type === "GitLab" || type === "Bitbucket") {
+        if (type === "GitLab" || type === "Bitbucket" || (type === "GitHub" && parsedUrl?.host !== "github.com")) {
```

Comment:
Feels like we should be able to pull out a function here called something like `isGithubEnterprise`.

Reply:
Good point, thanks! Done ✅